### ISSUE TITLE
toot tui crashes with Python 3.8

### ISSUE THREAD

**Issue by bill-auger** [ihabunek/toot#138]
```
$ toot tui
Traceback (most recent call last):
  File "/usr/bin/toot", line 11, in <module>
    load_entry_point('toot==0.24.0', 'console_scripts', 'toot')()
  File "/usr/lib/python3.8/site-packages/toot/console.py", line 545, in main
    run_command(app, user, command_name, args)
  File "/usr/lib/python3.8/site-packages/toot/console.py", line 527, in run_command
    return fn(app, user, parsed_args)
  File "/usr/lib/python3.8/site-packages/toot/commands.py", line 331, in tui
    TUI.create(app, user).run()
  File "/usr/lib/python3.8/site-packages/toot/tui/app.py", line 111, in run
    self.loop.run()
  File "/usr/lib/python3.8/site-packages/urwid/main_loop.py", line 286, in run
    self._run()
  File "/usr/lib/python3.8/site-packages/urwid/main_loop.py", line 384, in _run
    self.event_loop.run()
  File "/usr/lib/python3.8/site-packages/urwid/main_loop.py", line 1484, in run
    reraise(*exc_info)
  File "/usr/lib/python3.8/site-packages/urwid/compat.py", line 58, in reraise
    raise value
  File "/usr/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/lib/python3.8/site-packages/urwid/main_loop.py", line 172, in cb
    callback(self, user_data)
  File "/usr/lib/python3.8/site-packages/toot/tui/app.py", line 157, in <lambda>
    self.loop.set_alarm_in(0, lambda *args: done_callback(result))
  File "/usr/lib/python3.8/site-packages/toot/tui/app.py", line 259, in _done_initial
    self.timeline = self.build_timeline(timeline_name, statuses)
  File "/usr/lib/python3.8/site-packages/toot/tui/app.py", line 207, in build_timeline
    timeline = Timeline(name, statuses)
  File "/usr/lib/python3.8/site-packages/toot/tui/timeline.py", line 37, in __init__
    self.status_details = StatusDetails(statuses[focus], is_thread)
IndexError: list index out of range
```

**Comment by bill-auger** [comment]
i can not seem to reproduce this though - it could be a problem only on first run - after trying again the program is working fine

the tui is quite nice BTW - it even renders polls

**Comment by person-3** [comment]
I belive that this is caused by attempting to open the program whith an empty timeline

**Comment by ihabunek** [comment]
Fixed in bf168be87de51efc605c88935c0637b5e131c48e.
